### PR TITLE
EZR-3637: refactoring SDK Request classes and added docblocks

### PR DESCRIPTION
**Pull request by fatebinder**

### CODE REVIEW

File: src/Request/Model/Auth.php [hunk 1]
```diff
@@ -11,7 +11,14 @@
...
+     * @var integer
+     */
+    public $client_id;
+    /**
+     * @var string
+     */
+    public $client_secret;
 
     // Only used for grantType=refresh_token
     public $refresh_token;
```
Comment (pwarnes) [inline src/Request/Model/Auth.php]: Missing doc blocks... and below.

File: src/Response.php [hunk 2]
```diff
@@ -3,7 +3,7 @@
 namespace Easir\SDK;
 
 use GuzzleHttp\Psr7\Response as GuzzleResponse;
-use Easir\SDK\Model;
+//use Easir\SDK\Model;
```
Comment (pwarnes) [inline src/Response.php]: If this isn't needed it can be removed completely.

File: src/Model/LocalizationTypes.php [hunk 3]
```diff
@@ -17,7 +17,8 @@ class LocalizationTypes extends Response
             'currencies' => 'currency',
             'languages' => 'language',
             'countries' => 'country',
+            'options' => 'option',
```
Comment (pwarnes) [inline src/Model/LocalizationTypes.php]: What is options? This mapping tells the SDK that options is a collection of objects of type Option. Do we have an  Model of that name?

Comment (fatebinder) [inline src/Model/LocalizationTypes.php]: We do have one but I've not yet submitted that one, cuz the ticket we require it for is not finished (create leads ticket), so I propose to just remove them from here and I'll add them if we'll finish the other ticket

Comment (pwarnes) [inline src/Model/LocalizationTypes.php]: Agree. We don't want to merge something that's incomplete :)

**Pull request merged**